src/scss/override/_forms.scss

```diff
@@ -780,3 +780,46 @@ form .theme-light {
         }
     }
 }
+
+.putbox{
+	.icon{
+		float: left;
+		height: 68px;
+		width: 34px;
+		span{
+			display: block;
+			margin-top: 32px;
+			width: 34px;
+			height: 36px;
+			color: $color-accent;
+			font-size: 24px;
+			line-height: 36px;
+			text-align: left;
+		}
+	}
+	.box{
+		display: block;
+		margin-left: 34px;
+		min-height:32px;
+		color: $color-black;
+		.form-control{
+			margin: 0;
+		}
+		.btn-link{
+			margin: 0;
+			text-align: left;
+			padding-left: 0;
+			font-size: 14px;
+		}
+	}
+	.title{
+		font-size: 14px;
+		line-height: 32px;
+	}
+}
+
+.putbox-alt{
```

Comment:
This is an alt color theme, so can you please rename this class to `.theme-alt` instead of `.putbox-alt`? That's how Winstrap handles alt color themes. You can see [here ](https://github.com/winjs/winstrap/blob/master/src/scss/win/_buttons.scss#L141) for an example on how Winstrap does alt themes in other controls.